train.py

```diff
@@ -194,6 +194,9 @@ def main():
             print('WARNING: Initial learning rate (--lr) not set, selecting 0.1.')
         args.lr = 0.1
 
+    if args.generate_sample is not None and not args.act_mode_8bit:
+            print('WARNING: Cannot save sample in training mode, ignoring --save-sample option.')
```

Comment:
Please check the spaces (should be 4 spaces)

Reply:
Done.